api/src/main/java/org/openmrs/module/attachments/obs/BaseComplexData.java

```diff
@@ -28,8 +27,7 @@ public BaseComplexData(String title, Object data) {
 	}
 	
 	/**
-	 * This returns the byte array out of a complex data's inner data. This is borrowed from
-	 * {@link ComplexObsServlet}.
```

Comment:
Did you intentionally remove this? `This is borrowed from
	 * {@link ComplexObsServlet}`

Reply:
No @dkayiwa. Thanks for the thumbs up.